### ISSUE TITLE
Sobre nombre de atributos de unidad

### ISSUE THREAD

**Issue by notyourlasagna** [franciscogazitua/IIC2113-2024#161]
Hola! tengo una pregunta, el json que nos dieron tiene atributos como HP, Atk, Spd, Def, Res... estos, según el contexto, uno entiende lo que son, pero creo que unas clases atrás el profesor dijo que estaba mal si los usábamos así tal cual a la hora de hacerlos atributos de una clase por ejemplo. Con esto, me refiero respecto al nombre del atributo, por lo que quería corroborarlo. Por ejemplo, seremos penalizados en la E2 si una clase unidad tiene los atributos y queda: unidad.hp, unidad.atk, unidad.def... etc.? O debemos nombrarlos necesariamente unidad.HealthPoints, unidad.Attack... para tener todo el puntaje?

**Comment by RodrigoToroIcarte** [comment]
No, esos nombres son buenos dado su contexto. Ver https://github.com/franciscogazitua/IIC2113-2024/issues/40
